src/libraries/System.Private.CoreLib/src/System/Environment.Windows.cs

```diff
@@ -83,6 +84,34 @@ private static string ExpandEnvironmentVariablesCore(string name)
             return builder.ToString();
         }
 
+        private static unsafe bool IsAdminProcess()
```

Comment:
Since this is the implementation of IsPrivilegedProcess, should this instead be named something like GetIsPrivilegedProcess or IsPrivilegedProcessCore, rather than choosing a different word for Privileged?

Reply:
Changed to IsPrivilegedProcessCore.